ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/zuochunwei/velox.git
+VELOX_BRANCH=mydecimal
```

Comment:
Please change the branch

Reply:
Do you mean that change to `oap-project/main`?

Reply:
yes